### PR TITLE
Trasport proto impl

### LINKED ISSUE #190: [Transport protocol] Announce available content-type

### PR DESCRIPTION
**Pull request by sekulicd**
This implements transport(botd2) protocol.

This closes #190 

@tiero @altafan please review.

### CODE REVIEW

File: internal/interfaces/grpc/handler/transport.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	return &transportHandler{}
+}
+
+func (t transportHandler) SupportedContentTypes(
+	context.Context,
+	*transport.SupportedContentTypesRequest,
+) (*transport.SupportedContentTypesReply, error) {
+	return &transport.SupportedContentTypesReply{
+		AcceptedTypes: []transport.ContentType{
+			transport.ContentType_JSON,
```
Comment (altafan) [inline internal/interfaces/grpc/handler/transport.go:27]: Remove. At the moment the daemon does not support HTTP1 requests

**Review by altafan — CHANGES_REQUESTED** [review]
@sekulicd  we must update also the macaroon permissions.

We need to define a new entity `"transport"` and add the following permission in both `AllPermissionsByMethod()` and `Whitelist()`:

```

"/Transport/SupportedContentTypes": {{
	Entity: EntityTransport,
	Action: "read",
}},
```

File: internal/interfaces/grpc/handler/transport.go (hunk 1 above)
Comment (sekulicd) [reply internal/interfaces/grpc/handler/transport.go]: Done

**Comment by sekulicd** [comment]
> @sekulicd we must update also the macaroon permissions.
> 
> We need to define a new entity `"transport"` and add the following permission in both `AllPermissionsByMethod()` and `Whitelist()`:
> 
> ```
> 
> "/Transport/SupportedContentTypes": {{
> 	Entity: EntityTransport,
> 	Action: "read",
> }},
> ```

Done

**Pull request merged**